tests/bundle/EventListener/Admin/MainMenuBuilderListenerTest.php

```diff
@@ -22,7 +22,7 @@ final class MainMenuBuilderListenerTest extends TestCase
 {
     private MainMenuBuilderListener $listener;
 
-    private MockObject&AuthorizationCheckerInterface $authCheckerMock;
+    private AuthorizationCheckerInterface&MockObject $authCheckerMock;
```

Comment:
Is this done by CS fixer? If not, please do not turn this around. If it is, how did `SyliusEventDispatcherTest` manage to have it different?

Reply:
Yep :)